addons/l10n_fi/data/account.account.template.csv

```diff
@@ -229,3 +229,7 @@
 "account_9970","9970","Laskennalliset verot","account.data_account_type_expenses","l10n_fi.fi_chart_template","False","l10n_fi.account_tag_taxes_for_the_period"
 "account_9980","9980","Muut välittömät verot","account.data_account_type_expenses","l10n_fi.fi_chart_template","False","l10n_fi.account_tag_taxes_for_the_period"
 "account_9990","9990","Vähemmistöosuudet","account.data_account_type_expenses","l10n_fi.fi_chart_template","False",
+"account_19101","19101","Cash","account.data_account_type_liquidity","l10n_fi.fi_chart_template","False","l10n_fi.account_tag_liquidity"
```

Comment:
I really don't think we need this.

Reply:
No, we need at least one receivable/payable and a receivable for the PoS

Reply:
Outstanding receipts and payments accounts should be automatically created by the bank/cash journals when they are created at the installation of the chart template.